index.html

```diff
@@ -261,6 +261,8 @@ <h1>Glossary</h1>
                 <dd>The bounding box of an input (most commonly, touch) on a digitizer. This typically refers to devices with coarser pointer input resolution than a single pixel. Some devices do not report this data at all.</dd>
             <dt><dfn>digitizer</dfn></dt>
                 <dd>A type of input sensing device in which a surface can detect input which is in contact and/or in close proximity. Most commonly, this is the surface that senses input from the touch contact or a pen stylus.</dd>
+            <dt><dfn>direct manipulation</dfn></dt>
+                <dd>Certain input devices (such as touchscreens) implement a "direct manipulation" metaphor where a pointer is intended to act primarily on the UI element it became active upon (providing a physical illusion of direct contact, instead of indirect contact via a cursor that conceptually floats above the UI), and where the pointer not only serves to interact with controls, but is also used to directly pan, scroll or zoom the current page when it is active (e.g. when a finger or stylus touches the touchscreen).</dd>
```

Comment:
Let me try to reword your points and mention one difference from the indirect manipulation case for clarity (text selection):

> A pointer interaction metaphor where the input mechanism controlling the pointer (e.g. a finger or stylus touching a touchscreen) appears to be in a direct contact with an UI element on screen, and a cursor is not needed to mark the position of the pointer (unlike a mouse or trackpad).   Dragging direct manipulation pointers on screen usually cause a "sticky" effect with the UI elements under the pointers to convey the illusion of a direct physical contact, like panning/scrolling or zooming in such a way that the relative positions of the UI elements with respect to the pointers remain unchanged.  This is different from a mouse-drag (an indirect manipulation) that changes text selection highlight without moving the position of the UI element under the cursor.

Hmm, not sure if the use of "mechanism" vs "pointer" in my suggestion above look ambiguous enough!

Reply:
made another / similar stab at a rewording, by just giving some examples. i realise also that we're kind of using "direct manipulation" in a very narrow sense - as even some GUI interactions with a mouse are sometimes referred to as "direct manipulation" https://www.nngroup.com/articles/direct-manipulation/ ... which is why i tried to be a bit more direct and explicit here in my rewording by, essentially, calling a spade a spade and saying that touch/stylus are, but mouse isn't, for the purposes of our definition.

Reply:
Thanks for the reference, I now see the perspective behind the current definition.  I think this needs some discussion because IMHO this definition is a bit too broad to be able to differentiate between mouse and touch.

Let me summarize the two different definitions of "direct menipulation" we have mentioned here:

A. The nngroup page linked above defines "directness" exactly like [this Wikipedia page](https://en.wikipedia.org/wiki/Direct_manipulation_interface): if anything in the UI responds dynamically to a pointer, it qualifies as a direct manipulation, regardless of whether we have an explicit cursor (e.g. for mouse or trackpad nput) or not (onscreen touch or pen input).

B. In the issue discussion, the "directness" is like a classification of the pointing device: the input device can be "mouse-like" which is off-screen hence require an explicit cursor, or "touch-like" which comes in contact with the screen hence doesn't need a cursor.  In Chrome this is what we mean by direct manipulation.  I just discovered this ["guideline for developers"](https://docs.microsoft.com/en-us/previous-versions/windows/desktop/ms698539(v=vs.85)) from Microsoft which exactly matches our interpretation.

My mental model for the two deifnitions is as follow: the "direct manipulation" in [A] needs the UI target to stick to an explicit or implicit cursor, while in [B] the UI target needs to stick to the input device (apparently of course).  So it seems [A] covers all pointer input devices while [B] is only for "on-screen" devices.

Given the long-term impact our spec definition could possibly have on the developer community, we have to be extra careful with the term "direct menipulation".

Or maybe we can find an alternate term to avoid all the confusion?  It's still not too late!

Reply:
After our WG meeting discussion today, I am now convinced that we don't need to explicitly isolate "mouse" from `touch-action` and therefore sticking the broader definition of "direct manipulation" works fine.